external/graphdb-service.js

```diff
@@ -13,12 +13,13 @@ const constants = require('../modules/constants');
 class GraphdbService {
     constructor(config) {
         this.config = config;
+        this.config.url = `${this.config.ssl ? 'https' : 'http'}://${this.config.hostname}:${this.config.port}`;
```

Comment:
It can be done either by
- Deriving the url inside the constructor.
- Handing over the url to the constructor